CMakeLists.txt

```diff
@@ -806,6 +806,7 @@ if(BUILD_NVFUSER_BENCHMARK)
   )
   target_include_directories(nvfuser_bench PUBLIC ${NVFUSER_ROOT})
   target_link_libraries(nvfuser_bench PRIVATE
+    GTest::gtest
```

Comment:
Including tests/cpp/utils.cpp in source files without linking gtest was a bug. This change fixes it.